cmd/tk/env.go

```diff
@@ -248,6 +249,7 @@ func envListCmd() *cli.Command {
 		if err != nil {
 			return err
 		}
+		sort.Slice(envs, func(i, j int) bool { return envs[i].Metadata.Name < envs[j].Metadata.Name })
```

Comment:
```suggestion
		sort.SliceStable(envs, func(i, j int) bool { return envs[i].Metadata.Name < envs[j].Metadata.Name })
```

In case there are equal elements in the list, Slice is not guaranteed to be deterministic